src/environment.rs

```diff
@@ -326,6 +327,13 @@ impl Environment {
         environment.insert(Symbol::intern("defmacro"), defmacro_macro.to_rc_value());
         environment.insert(Symbol::intern("eval"), eval_fn.to_rc_value());
 
+        // Interopto read real clojure.core
+        environment.insert_into_namespace(
+            &Symbol::intern("clojure.interop"),
```

Comment:
Instead of removing this PR, I suppose we can just make some changes

Basically, as touched on,  `count` should be `intern("clojure.core","count")` , and the rest will work automatically

Reply:
done